src/creature.h

```diff
@@ -91,14 +91,22 @@ class Creature
         /**
          * Simplified attitude towards any creature:
          * hostile - hate, want to kill, etc.
-         * friendly - avoid harming it, maybe even help.
          * neutral - anything between.
+         * friendly - avoid harming it, maybe even help.
+         * any - any of the above, used in safemode_ui
          */
-        enum Attitude {
+        enum Attitude : int {
             A_HOSTILE,
             A_NEUTRAL,
             A_FRIENDLY,
+            A_ANY
```

Comment:
This is a bit strange. "Any" is not a valid return value of `attitude_to`, it's only used by the safemode ui. Making it part of this enumeration breaks encapsulation, code that inspects the result of `attitude_to` now has to consider `A_ANY` as well (through it will never be returned by `attitude_to`).

And you need special handling for it in the safemode ui anyway, so you can just add another boolean property to the rules there and use that to indicate "any attitude".